ironfish/src/fileStores/config.ts

```diff
@@ -315,6 +329,11 @@ export const ConfigOptionsSchema: yup.ObjectSchema<Partial<ConfigOptions>> = yup
     networkId: yup.number().integer().min(0),
     customNetwork: yup.string().trim(),
     maxSyncedAgeBlocks: yup.number().integer().min(0),
+    mempoolMaxSizeBytes: yup
+      .number()
+      .integer()
+      .min(2 * MEGABYTES), // one blocks worth of transactions
```

Comment:
This should be using `consensus.maxBlockSizeBytes`, so we don't have to change multiple places if we support variable block sizes.

It's not very easy to access that in the config though, so I wonder if it might be better to have the config min at 0, then where `mempoolMaxSizeBytes` is used, use the max of `maxBlockSizeBytes, mempoolMaxSizeBytes`. Alternatively, is it that much of an issue if someone sets a very small mempool size? They'll probably have to spend a bit more on bandwidth fetching missing transactions

Reply:
I remember we had this discussion and decided to put this config value in terms of straight bytes instead of blocks so it's makes more sense to users (they can see a real byte value). I will remove this comment connecting it to block size